docs/configuring.md

```diff
@@ -2,36 +2,66 @@
 sidebar_position: 3
 ---
 
-# Configuring
+# Preferences
 
-There are two approaches to configuring Pixeebot:
+Pixeebot works immediately after installation without any further configuration. Yet, Pixeebot exposes preferences for users to tailor its behavior when desirable.
 
-1. **Target repository configuration:**
-   Create a `pixeebot.yaml` file in the `.github` directory of the target repository. Configurations in the target repository will take precedence over other configurations.
+Users may set Pixeebot preferences at either the organization or repository levels.
 
-2. **Global repository configuration:**
-   Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global configuration that applies to multiple repositories.
+1. **Repository Preferences:**
+   Create a `pixeebot.yaml` file in the `.github` directory of the target repository. Preferences in the target repository will take precedence over other preferences.
 
-## YAML
+2. **Organization-Wide preferences:**
+   Alternatively, you can create the `pixeebot.yaml` file in the `.github` directory of your `.github` repository. This will serve as a global preferences that applies to multiple repositories.
```

Comment:
"This will serve as a global preference..." (singular)

Reply:
Ok then I would say "This will serve as (the?) global preferences that apply to multiple repositories.

Reply:
How do you feel about replacing the "multiple repositories" with a more specific "all repositories in this installation"? I feel like "multiple repositories" is ambiguous. Which repositories? At Pixeebot's discretion?!

Reply:
Yes I like that, we say something similar in the View/change preference modal in the UI.